ci-scripts/rhdh-setup/template/backstage/chart-values.yaml

```diff
@@ -84,6 +84,37 @@ upstream:
         value: "warn"
       - name: NODE_TLS_REJECT_UNAUTHORIZED
         value: "0"
+    extraContainers:
+      - name: oauth2-proxy
+        env:
+          - name: OAUTH2_PROXY_CLIENT_ID
+            valueFrom:
+              secretKeyRef:
+                key: CLIENT_ID
+                name: keycloak-client-secret-backstage
+          - name: OAUTH2_PROXY_CLIENT_SECRET
+            valueFrom:
+              secretKeyRef:
+                key: CLIENT_SECRET
+                name: keycloak-client-secret-backstage
+          - name: OAUTH2_PROXY_COOKIE_SECRET
+            value: '${COOKIE_SECRET}'
```

Comment:
@yogananth-subramanian A best-practice advice: As it is a secret, it should not be put into the container spec as plain text. You should put it into a `Secret` resource and use the same `.valueFrom.secretKeyRef` approach as shown in case of `CLIENT_SECRET` variable above.

Reply:
Thanks @pmacik  for the review, agree with your comment. Have fixed it in the latest commit, have moved it to a secret.